22_to_do_list/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Pacifico&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>To do list</title>
+  </head>
+  <body>
+    <header class="header"><h1>To do list</h1></header>
+    <main>
+      <div class="alert visuallyhidden js-alert">
+        <p>Please enter a new task fisrt</p>
+      </div>
+      <form class="flex-input" action="">
+        <label for="task" class="visuallyhidden">Input task</label>
+        <input type="text" name="task" id="task" />
+        <button class="btn-add js-btn-add">Add Item</button>
+      </form>
+      <ul class="flex-list js-list">
```

Comment:
I'm looking forward to seeing you create a todo list using a modern UI Framework! (react, vue, etc...) :D